### ISSUE TITLE
Load Message

### ISSUE THREAD

**Issue by epicstar** [rectangle-dbmi/Realtime-Port-Authority#216]
For every major app version change, we will have a one-time dialog popup saying what changed on the update. Linked with #215

**Comment by sgdoerfler** [comment]
When I switch to the app, I'm probably focused on catching a bus. I'd rather read about changes later on, when it's more convenient. So a one-time popup dialog is fine, but it would be good to have a way to review changes later on too.

Perhaps the About dialog, which already links to the project's Github site, could have a further direct link to https://github.com/rectangle-dbmi/Realtime-Port-Authority/releases labeled "Release Notes" or "Version History", and that page could be updated with a new entry for each major app version change?

**Comment by epicstar** [comment]
Good idea... Maybe this method is better. I'll discuss with @VincentIII.

**Comment by epicstar** [comment]
@sgdoerfler -> your wish has been granted :+1:
